client/src/App.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-
+import SignIn from './pages/SignIn';
 function App() {
   return (
     <div className="App">
-      START
+      <SignIn></SignIn>
```

Comment:
```suggestion
      <SignIn />
```
자식 안받으니 Self-Closing으로 해도 될듯?